scio-google-cloud-platform/src/main/scala/com/spotify/scio/bigquery/BigQueryIO.scala

```diff
@@ -68,30 +68,18 @@ private object Reads {
     flattenResults: Boolean = false
   ): SCollection[T] = {
     val bigQueryClient = client(sc)
-    if (bigQueryClient.isCacheEnabled) {
-      val read = bigQueryClient.query
-        .newQueryJob(sqlQuery, flattenResults)
-        .map { job =>
-          sc.onClose(_ => bigQueryClient.waitForJobs(job))
-          typedRead.from(job.table).withoutValidation()
-        }
-
-      sc.applyTransform(read.get)
-    } else {
-      val baseQuery = if (!flattenResults) {
-        typedRead.fromQuery(sqlQuery).withoutResultFlattening()
-      } else {
-        typedRead.fromQuery(sqlQuery)
-      }
-      val query = if (bigQueryClient.query.isLegacySql(sqlQuery, flattenResults)) {
-        baseQuery
-      } else {
-        baseQuery.usingStandardSql()
+    val labels = sc.labels
```

Comment:
Are we removing the cache flag here? The `else` branch is in case if the user disables cache.

Reply:
The BQ client already supports it.